external/LibraryThemeStyles/src/Controller/AdminController.php

```
@@ -39,24 +29,27 @@ public function indexAction()
-                    $error = 'Unknown action.';
+                // Collect form data
+                $data = [
+                    'action' => $this->params()->fromPost('action'),
+                    'target_preset' => $this->params()->fromPost('target_preset', 'modern'),
+                    'site' => $siteSlug,
+                    'debug' => false, // Admin interface doesn't need debug mode
+                ];
+
+                // Delegate to ModuleConfigService for consistent handling
+                $messenger = $this->messenger();
+                $success = $this->moduleConfigService->handleConfigFormSubmission($data, $messenger);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$success'. (undefined)

(UnusedLocalVariable)